src/Uno.UI.Tests/App/Xaml/Test_Control.xaml.cs

```diff
@@ -25,8 +26,26 @@ public sealed partial class Test_Control : UserControl
 		public Grid TopGrid => topGrid;
 		public CheckBox TestCheckBox => testCheckBox;
 		public TextBlock TestTextBlock => testTextBlock;
+		public TextBlock TestTextBlock2 => testTextBlock2;
+		public Border TestBorder => testBorder;
+		public Grid TestGrid => testGrid;
+		public Button StyledButton => styledButton;
+		public MyControl TestMyControl => testMyControl;
+		public ResourceTestControl InlineTemplateControl => inlineTemplateControl;
+		public ResourceTestControl TemplateFromResourceControl => templateFromResourceControl;
+		public RadioButton TestRadioButton => testRadioButton;
+		public RadioButton TestRadioButtonExplicit => testRadioButtonExplicit;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'TestRadioButtonExplicit' a static property.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826539)

Reply:
That... would not work